docs/developer-guide/local_testing_guide.md

```
@@ -0,0 +1,285 @@
+$ tar -xzf go1.12.1.linux-amd64.tar.gz
+$ PATH=/home/$USER/src/golang/go1-12-1/go/bin:$GOPATH/src/k8s.io/kubernetes/_output/local/bin/linux/amd64:$PATH
+```
+
+### Download and Build Kubernetes Components (E2E Tests, ginkgo, kubectl):
+
+Determine which version of Kubernetes is currently used in CI (See
+[ovn-kubernetes/.github/workflows/test.yml](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml))
+and set the environmental variable `K8S_VERSION` to the same value. Also make sure to export a GOPATH which points to
+your go directory with `export GOPATH=(...)`.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)